src/en/news/blog/2023/v16-2-14-pacific-released/index.md

```diff
@@ -0,0 +1,388 @@
+
+- qa: run scrub post file system recovery ([issue#59527](http://tracker.ceph.com/issues/59527), [pr#51610](https://github.com/ceph/ceph/pull/51610), Venky Shankar)
+
+- qa: test\_simple failure ([pr#50756](https://github.com/ceph/ceph/pull/50756), Patrick Donnelly)
+
+- qa: use parallel gzip for compressing logs ([pr#52953](https://github.com/ceph/ceph/pull/52953), Patrick Donnelly)
+
+- qa: wait for MDSMonitor tick to replace daemons ([pr#52237](https://github.com/ceph/ceph/pull/52237), Patrick Donnelly)
+
+- radosgw-admin: try reshard even if bucket is resharding ([pr#51836](https://github.com/ceph/ceph/pull/51836), Casey Bodley)
+
+- rbd-mirror: fix image replayer shut down description on force promote ([pr#52878](https://github.com/ceph/ceph/pull/52878), Prasanna Kumar Kalever)
+
+- rbd-mirror: fix race preventing local image deletion ([pr#52625](https://github.com/ceph/ceph/pull/52625), N Balachandran)
+
+- rgw/rados: check\_quota() uses real bucket owner ([pr#51330](https://github.com/ceph/ceph/pull/51330), Mykola Golub, Casey Bodley)
+
+- rgw/s3: dump Message field in Error response even if empty ([pr#51200](https://github.com/ceph/ceph/pull/51200), Casey Bodley)
+
+- rgw: avoid string\_view to temporary in RGWBulkUploadOp ([pr#52159](https://github.com/ceph/ceph/pull/52159), Casey Bodley)
+
+- rgw: fix consistency bug with OLH objects ([pr#52552](https://github.com/ceph/ceph/pull/52552), Cory Snyder)
+
+- rgw: LDAP fix resource leak with wrong credentials ([pr#50560](https://github.com/ceph/ceph/pull/50560), Johannes Liebl, Johannes)
+
+- rgw: under fips & openssl 3<span></span>.x allow md5 usage in select rgw ops ([pr#51266](https://github.com/ceph/ceph/pull/51266), Mark Kogan)
+
+- src/valgrind<span></span>.supp: Adding know leaks unrelated to ceph ([pr#49521](https://github.com/ceph/ceph/pull/49521), Nitzan Mordechai)
+
+- src/valgrind<span></span>.supp: Adding know leaks unrelated to ceph ([pr#51341](https://github.com/ceph/ceph/pull/51341), Nitzan Mordechai)
+
+- test: correct osd pool default size ([pr#51803](https://github.com/ceph/ceph/pull/51803), Nitzan Mordechai)
+
+- test: monitor thrasher wait until quorum ([pr#51799](https://github.com/ceph/ceph/pull/51799), Nitzan Mordechai)
+
+- tests: remove pubsub tests from multisite ([pr#48928](https://github.com/ceph/ceph/pull/48928), Yuval Lifshitz)
+
+- tools/ceph-dencoder: Fix incorrect type define for trash\_watcher ([pr#51778](https://github.com/ceph/ceph/pull/51778), Chen Yuanrun)
+
+- tools/ceph-kvstore-tool: fix segfaults when repair the rocksdb ([pr#51254](https://github.com/ceph/ceph/pull/51254), huangjun)
+
+- tools/cephfs-data-scan: support for multi-datapool ([pr#50523](https://github.com/ceph/ceph/pull/50523), Mykola Golub)
+
+- v16<span></span>.2<span></span>.13 ([pr#51404](https://github.com/ceph/ceph/pull/51404), Ceph Release Team, Patrick Donnelly, Rishabh Dave, Casey Bodley, Radoslaw Zarzynski, Mark Nelson)
```

Comment:
I don't think we want this, right @neha-ojha?

Reply:
Meaning line 383

Reply:
@ljflores I removed it